### ISSUE TITLE
Import not working at all

### ISSUE THREAD

**Issue by tynana** [AMoreaux/Sketch-Icons#69]
When I click "Import icons" nothing happens. I work on Sketch 70.5, MacOS Big Sur with M1 processor. 
Please fix this, your plugin was such a time saver 🙏🏻

**Comment by montemartin** [comment]
I'm in the same boat, Sketch 71.2, Big Sur and M1. I was really excited to save _hours_ arranging an icon library I'm working on. I know the plugin is working - I can access the settings modal, and if I run organize or replace without a layer selected an alert warns me to select a layer, but once a layer is selected nothing other than settings generates response.

I assume it has something to do with with the M1 processor. I had issues getting Runner working until they pushed out an update.

**Comment by mauriziomambrini** [comment]
I have the same problem, with Sketch 73.1, Big Sur and M1. Please fix this! 🙏🏻

**Comment by AMoreaux** [comment]
Hi there,
I have no access to a mac M1 so I can't debug this issue. The project is open, if anyone wants to create a PR, I will be happy to work with him.

**Comment by oliverkierepka** [comment]
Sketch Import does not work. It does not open the popup at all.
![image](https://user-images.githubusercontent.com/552528/151361402-0bc47b0f-2a2d-49f6-b5ef-29fd6075fc6a.png)

**Comment by hbxes** [comment]
Yes I have the same issue, i can't import. If I click on "Import" on the plugin, then nothing happens.  (I work with an Apple M1 Pro, Sketch Version 82.1. It would be nice to fix it soon.

**Comment by james182** [comment]
I have sketch 75, on the new MBP M1 Max and this plugin is not working, nothing happens. I love this plugin it saves soooo much time. Please Fix.

**Comment by hbxes** [comment]
For the MAC with M1 Chip there is a workaround for the moment: 

You have to start the Sketch App with **Rosetta**, then the plug-in works.

Got to the Applications folder, right click on sketch app, then select info (information in german) and tick "open with rosetta"

Hope that helps for the moment.  But still a real fix would be appreciated

<img width="216" alt="Bildschirmfoto 2022-03-23 um 09 23 21" src="https://user-images.githubusercontent.com/55880422/159655150-a0fb40e7-dd63-4d28-a9cc-3ffc56b62925.png">

**Comment by oliverkierepka** [comment]
> 

Thank you :) This helped

**Comment by oliverkierepka** [comment]
> I have sketch 75, on the new MBP M1 Max and this plugin is not working, nothing happens. I love this plugin it saves soooo much time. Please Fix.

@james182   Thanks, this helped :)

**Comment by maryroe** [comment]
I have the same issue with Macbook M1. Thank you for creating one of the best plugins

**Comment by joshsanders** [comment]
running into this same issue ... currently on macOS Monterey v12.2.1 , Apple M1, Sketch v89 .  Loved this plugin too...one of my most used. 😩

**Comment by MickL** [comment]
1.5 years later, still not working :( I tried to open Sketch (v99) with Rosetta but when I click import icons the plugin crashes.

@AMoreaux is this project still maintained?

**Comment by Duffdaddy** [comment]
Tried it June 2026 in current version of Sketch, opened under Rosetta as suggested and it does open an appear to run. You get a notification its completed it's import, but nothing every appears on your canvas, page or frame. 
Seems to be non functional in its current state with current versions of Sketch on OSX.
